apps/concierge_site/lib/endpoint.ex

```diff
@@ -17,7 +17,7 @@ defmodule ConciergeSite.Endpoint do
     at: "/",
     from: :concierge_site,
     gzip: true,
-    only: ~w(css fonts images js favicon.ico robots.txt)
+    only_matching: ~w(css fonts images js favicon robots)
```

Comment:
TIL

Reply:
Same! The docs for this are surprisingly vague. I had to dig into the source to even understand that what it was "matching" against (in both cases) is just the first path component.